activiti-services/activiti-services-rest/src/main/java/org/activiti/services/rest/controllers/TaskVariableController.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.activiti.services.rest.controllers;
+
+import java.util.Map;
+
+import org.activiti.engine.TaskService;
+import org.activiti.services.core.ProcessEngineWrapper;
+import org.activiti.services.core.model.TaskVariables;
+import org.activiti.services.core.model.TaskVariables.TaskVariableScope;
+import org.activiti.services.core.model.commands.SetTaskVariablesCmd;
+import org.activiti.services.rest.resources.assembler.TaskVariableResourceAssembler;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.hateoas.MediaTypes;
+import org.springframework.hateoas.Resource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/v1/{taskId}/variables",
```

Comment:
@daisuke-yoshimoto ouch sorry my bad, this should be /v1/tasks/{taskId}/variables right? it needs to fall under the task URL path to be consistent with what the process instances variable endpoint. @erdemedeiros can you please check this as well.